plugins/c8y_configuration_plugin/src/topic.rs

```diff
@@ -1,26 +1,22 @@
 use mqtt_channel::TopicFilter;
 
 #[derive(Debug, Clone, Eq, PartialEq)]
-pub enum ConfigResponseTopic {
+pub enum ConfigOperationTopic {
     SnapshotResponse,
     SnapshotRequest,
```

Comment:
Why was `SnapshotRequest` retained when `UpdateRequest` was removed? Either both should be there, else neither should be there. The enum name `ConfigOperationTopic` is appropriate if both request and response variants are present in this single enum. But, if you're going to have separate enums for requests and responses, `ConfigOperationRequestTopic` and `ConfigOperationResponseTopic` would be better names.